### PR TITLE
Fixes to WCAG 2.1 Understanding 2.4.6 and 3.3.2

### PR DESCRIPTION
**Pull request by patrickhlauke**
Closes https://github.com/w3c/wcag/issues/610

(note I wasn't sure whether to make the fixes on these files, which are in the 2.0 folder, or whether or not I should have created new instances of these in the 2.1 folder)

### CODE REVIEW

File: understanding/20/headings-and-labels.html [hunk 1]
```diff
@@ -26,46 +26,43 @@ <h2>Intent of Headings and Labels</h2>
...
-            <p>This Success Criterion may also help users with low vision who can see only a few
-               words at a time. 
-               
-            </p>
-            
-         </li>
-         
+
+         <li>When headings and labels are also correctly marked up and identified in accordance with <a href="info-and-relationships">1.3.1: Info and Relationships</a>, this Success Criterion helps people who use screen readers by ensuring that labels and headings are meaningful when read out of context, for example, in an automatically generated list of headings/table of contents, or when jumping from heading to heading within a page.</li>
+
```
Comment (awkawk) [inline understanding/20/headings-and-labels.html]: <li>Form input controls with labels that clearly describe the content that is expected to be input helps users know how to successfully complete the form.</li>

Comment (patrickhlauke) [reply understanding/20/headings-and-labels.html]: > * Form input controls with labels that clearly describe the content that is expected to be input helps users know how to successfully complete the form.

good suggestion, added

File: understanding/20/labels-or-instructions.html [hunk 2]
```diff
@@ -24,47 +23,44 @@ <h2>Intent of Labels or Instructions</h2>
...
-         </p>
          
-      </div>
-      
-      
+      <p>This Success Criterion does not require that labels or instructions be correctly marked up or
+         identified/associated with their respective controls - this aspect is covered separately by
+         <a href="info-and-relationships">1.3.1: Info and Relationships</a>. It is possible for content
+         to pass this Success Criterion (providing relevant labels and instructions) while failing
+         Success Criterion 1.3.1 (if the labels or instructions aren't correctly marked up/identified/associated).
```
Comment (awkawk) [inline understanding/20/labels-or-instructions.html]: I would like to avoid the "this/this/this" construction

Comment (awkawk) [reply understanding/20/labels-or-instructions.html]: maybe just separate with commas

File: understanding/20/labels-or-instructions.html [hunk 3]
```diff
@@ -24,47 +23,44 @@ <h2>Intent of Labels or Instructions</h2>
...
+      </p>
+      <p>Further, this Success Criterion does not take into consideration whether or not alternative methods of
+         providing an accessible name or description for form controls and inputs has been used - this aspect is
+         covered separately by <a href="name-role-value">4.1.2: Name, Role and Value</a>. It is possible
+         for controls and inputs to have an appropriate accessible name or description (e.g. using <code>aria-label="..."</code>)
+         and therefore pass Success Criterion 4.1.2, but to still fail this Success Criterion (if the labels or instructions
+         aren't presented to all users, not just those using assistive technologies).
+      </p>
+      <p>While this Success Criterion <em>requires</em> the use of labels for controls and inputs, whether or not
```
Comment (awkawk) [inline understanding/20/labels-or-instructions.html]: I'm not sure why requires is em'd here. I also think that it makes it sound more like the label element to me in this phrasing. I'd prefer "While this Success Criterion requires that controls and inputs have labels, whether or not"

File: understanding/20/labels-or-instructions.html [hunk 4]
```diff
@@ -24,47 +23,44 @@ <h2>Intent of Labels or Instructions</h2>
...
-            readers when the field receives focus and users with impaired motor control are helped
-            by a larger clickable area for the control, since clicking on the label or the control
-            will activate the control.
-         </li>
-         
-         <li>Field labels located in close proximity to the associated field assist users of screen
-            magnifiers because the field and label are more likely to visible within the magnified
-            area of the page.
+         <li>Providing clear and unambiguous labels and instructions (including examples of expected
+            data formats) helps all users - but particularly those with cognitive, language and learning
```
Comment (awkawk) [inline understanding/20/labels-or-instructions.html]: we use the serial comma :)

File: understanding/20/labels-or-instructions.html (hunk 3 above)
Comment (patrickhlauke) [reply understanding/20/labels-or-instructions.html]: fixed

File: understanding/20/labels-or-instructions.html (hunk 4 above)
Comment (patrickhlauke) [reply understanding/20/labels-or-instructions.html]: fixed

File: understanding/20/labels-or-instructions.html (hunk 2 above)
Comment (patrickhlauke) [reply understanding/20/labels-or-instructions.html]: not quite clear what you mean. the use of "this" in that one paragraph? or the fact there are three paragraphs starting with "This...", "Further, this..." and "While this..."?

Comment (awkawk) [reply understanding/20/labels-or-instructions.html]: "(if the labels or instructions aren't correctly marked up, identified, or associated)"

Comment (patrickhlauke) [reply understanding/20/labels-or-instructions.html]: ah, gotcha. fixed and force-pushed an update

**Comment by awkawk** [comment]
@patrickhlauke The WG approves the direction, although @mbgower will have some input on the pull request. :)

**Pull request merged**